_posts/2023-02-16-secure-cloud-foundation.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: post
+authors: [lander_marien, duncan_casteleyn]
+title: 'Secure cloud foundation tooling'
+image: /img/2023-02-16-secure-cloud-foundation/header.png # TODO - change image
+tags: [aws, cloud]
+category: Cloud
+comments: true
+---
+
+- [Introduction](#introduction)
+- [The Warden: Open Policy Agent](#the-warden-open-policy-agent)
+- [The Janitor: Cloud Custodian](#the-janitor-cloud-custodian)
+- [The Watchdog: Snyk](#the-watchdog-snyk)
+- [The Container Detective: Trivy](#the-container-detective-trivy)
+- [The Composer: Fugue](#the-composer-fugue)
+
+# Introduction
+
+If you ever came in contact with an enterprise cloud environment,
+you know that keeping it secure and compliant can be a challenging task.
+
+Every company has their own security and compliance requirements,
+and they should at least follow a best practice standard like CIS AWS Foundations Benchmark.
+
+Often, you'll find that because of the segmented responsibilities across many teams and projects, gaps exist in the security and compliance posture of the organisation. 
+Add to that the rate of change that companies need to handle to stay competitive and you end up with a puzzle that no-one can oversee. 
+
+Originally, we started investigating solutions to scan your infrastructure as code and cloud environment.
+We thought we'd look at a handful of tools, evaluate them and select a clear winner.
+
+We quickly discovered there are quite some interesting tools out their,
+some that are free and some that cost you a lot of money,
+and some not only do cloud, but can do much more.
+
+We'll be summarizing the tools we found and how they can help you to secure your cloud environment or other resources.
+What they can do, what they can't do, what you can use them with and for.
+
+## The Warden: Open Policy Agent
+
+Cloud computing has made it easy for organizations to manage their IT infrastructure on a large scale. However, with the ease of cloud computing comes the challenge of securing the environment and preventing accidental misconfigurations. This is where [Open Policy Agent](https://www.openpolicyagent.org/){:target="_blank" rel="noopener noreferrer"} (OPA) comes into play.
+
+<img src="{{ '/img/2023-02-16-secure-cloud-foundation/warden.png' | prepend: site.baseurl }}" alt="Gandalf, You shall not pass" class="image fit" style="margin:0px auto; max-width:100%">
+
+OPA is an open-source, general-purpose policy engine that can be used to enforce policies across various systems, including cloud infrastructure.
+
+Some of OPA's strenghts summed up:
```

Comment:
```suggestion
Some of OPA's strengths summed up:
```

Reply:
Actually, I would rewrite this sentence, the ':' is not that great. 
A suggestion:
'Lets look at some of the strengths and advantages that OPA has to offer.'